nuttx/arch/arm/src/stm32/Kconfig

```diff
@@ -1235,156 +1926,166 @@ if STM32_ADC1
 menu "ADC1 Device Initialization"
 
 config STM32_ADC1_CHAN0_INIT
-	bool "ADC1 Channel 0"
+	bool "ADC1 Channel 0 (IN1-PC0)"
```

Comment:
Hi CaveMike,
     I don't currently have the data to confirm chipsets outside the STM32L4 series, but the config scripts have not changed with the exception to adding the INx designation and the addition of "depends on" conditions within the ADC 1 to 3 lists that trim the list based on STM32L4 type and package size.   The following list the ADC delta between v476 and v433/31.  

Port	ADC for v476	ADC for v433/31
PA0	ADC12_IN5	ADC1_IN5
PA1	ADC12_IN6	ADC1_IN6
PA2	ADC12_IN7	ADC1_IN7
PA3	ADC12_IN8	ADC1_IN8
PA4	ADC12_ IN9	ADC1_IN9
PA5	ADC12_ IN10	ADC1_IN10
PA6	ADC12_ IN11	ADC1_IN11
PA7	ADC12_ IN12	ADC1_IN12
PB0	ADC12_ IN15	ADC1_IN15
PB1	ADC12_ IN16	ADC1_IN16
PC0	ADC123_ IN1	ADC1_IN1
PC1	ADC123_ IN2	ADC1_IN2
PC2	ADC123_ IN3	ADC1_IN3
PC3	ADC123_ IN4	ADC1_IN4
PC4	ADC12_ IN13	ADC1_IN13
PC5	ADC12_ IN14	ADC1_IN14
PF3	ADC3_IN6	
PF4	ADC3_IN7	
PF5	ADC3_IN8	
PF6	ADC3_IN9	
PF7	ADC3_IN10	
PF8	ADC3_IN11	
PF9	ADC3_IN12	
PF10	ADC3_IN13